bom/application/pom.xml

```diff
@@ -123,8 +123,7 @@
         <postgresql-jdbc.version>42.3.3</postgresql-jdbc.version>
         <mariadb-jdbc.version>2.7.5</mariadb-jdbc.version>
         <mysql-jdbc.version>8.0.28</mysql-jdbc.version>
-        <mssql-jdbc.version>7.2.2.jre8</mssql-jdbc.version>
-        <adal4j.version>1.6.7</adal4j.version>
```

Comment:
You need to remove the `<dependency>` using this property in this file

Reply:
I did. Did not help the integration test failure...

Reply:
Maybe you can try on your machine. Could be something with mine...